scripts/pages/factsheet-place.page.js

```diff
@@ -26,25 +27,34 @@ const defaults = {
 };
 
 const _build = data => {
+  const renderCallback = function() {
+    this.el.parentElement.classList.toggle('is-hidden', !this.el.parentElement.classList.contains('is-hidden'));
+  };
+
   new Line('.js-line', data.trajectory_deforestation, data.trajectory_production);
   new Chord('.js-chord-traders', data.top_traders_matrix, data.top_traders, data.municip_name);
   new Chord('.js-chord-consumers', data.top_consumers_matrix, data.top_consumers, data.municip_name);
-  new Table({
-    el:document.querySelector('.js-table-traders'),
-    data: data.top_traders, // example
-    type: 'top',
-    target: 'actor'
-  });
 
-  new Table({
-    el:document.querySelector('.js-table-consumers'),
-    data: data.top_consumers, // example
-    type: 'top'
-  });
+
+  new Top({
+    el: document.querySelector('.js-top-trader'),
+    data: data.top_traders,
+    targetLink: 'actor',
+    title: 'top traders',
+    unit: '%'
+  }).render(renderCallback);
```

Comment:
I still don't see the point. You do :
```
new Top (...)
new Top (...)
this.el.parentElement.classList.toggle (...)
```
it's functionally the same without the unnecessary tight coupling between the component and its parent.